charts/piraeus/templates/config.yaml

```diff
@@ -54,3 +54,7 @@ data:
             image: drbd9-focal
           - osImage: Ubuntu 22\.04
             image: drbd9-jammy
+          - osImage: Debian GNU/Linux 11 (bullseye)
+            image: drbd9-bullseye
+          - osImage: Debian GNU/Linux 10 (buster)
```

Comment:
I'd use a more general pattern, since these are regular expressions anyways. Perhaps just `bullseye` and `buster` are enough?

Reply:
I'd be more inclined to use Debian GNU/Linux 11 and strip away the codename in brackets. Not fully sure how Kustomize handles it.

Reply:
> strip away the codename

Also fine by me. It should "just work".